CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ release lines.
 
 Select a Node.js version below to view the changelog history:
 
-* [Node.js 10](doc/changelogs/CHANGELOG_V10.md) — **Current**
+* [Node.js 11](doc/changelogs/CHANGELOG_V11.md) - **Current**
+* [Node.js 10](doc/changelogs/CHANGELOG_V10.md) — Current
```

Comment:
Are we going to have two current releases (10 and 11) until 10 turns LTS later this month?

Reply:
Yes, unfortunately.